example-get-started-experiments/code/.github/workflows/deploy-model.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy model
+
+on:
+  push:
+    # When a new version is registered in Studio Model Registry
+    tags:
+      - "results/train=pool-segmentation#*"
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Manual version name'
+        required: true
+        type: string
+  
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  parse:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: "Parse GTO tag"
+      id: gto
+      uses: iterative/gto-action@14723404a00bb0c1e759c02ffcd24279df5815c2
+    outputs:
+      event: ${{ steps.gto.outputs.event }}
+      name: ${{ steps.gto.outputs.name }}
+      stage: ${{ steps.gto.outputs.stage }}
+      version: ${{ steps.gto.outputs.version }}
+
+  deploy-model:
+    needs: parse
+    if: ${{ needs.parse.outputs.event }} == 'assignment'
+    environment: cloud
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: aws-actions/configure-aws-credentials@v2
+      with:
+        aws-region: us-east-2
+        role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+        role-duration-seconds: 43200
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+        cache: 'pip'
+        cache-dependency-path: requirements.txt
+
+    - run: pip install -r requirements.txt
+
+    - run:  dvc remote add -d --local storage s3://dvc-public/remote/get-started-pools
+
+    - run: |
+        MODEL_DATA=$(dvc get --show-url . model.tar.gz)
```

Comment:
@daavoo Doesn't this get the model data for the latest commit instead of the specific version we are trying to deploy? From the DVC docs, when `--rev` is not specified, `The latest commit (in the default branch) is used by default when this option is not specified.`

Reply:
I assume that the phrase in the docs is meant for the scenario where you pass a remote URL repo as the first argument. 

I think the behavior here is correct, but I am going to double-check.

I think it works because we are using the local repo `.` and so the current status of the workspace will be used. The workflow uses `actions/checkout` and runs on the git tag creation so the workspace will be the status of the revision where the tag was created